test/test_lp.jl

```diff
@@ -1,175 +1,176 @@
 using Convex
-using FactCheck
+using Test
+import Random.shuffle
 
 TOL = 1e-3
 
-facts("LP Atoms") do
+@testset "LP Atoms" begin
 
-  context("abs atom") do
+  @testset "abs atom" begin
     x = Variable()
     p = minimize(abs(x), x<=-1)
-    @fact vexity(p) --> ConvexVexity()
+    @test vexity(p) == ConvexVexity()
     solve!(p)
-    @fact p.optval --> roughly(1, TOL)
-    @fact evaluate(abs(x)) --> roughly(1, TOL)
+    @test p.optval ≈ 1 atol=TOL
+    @test evaluate(abs(x)) ≈ 1 atol=TOL
 
     x = Variable(2,2)
     p = minimize(sum(abs(x)), x[2,2]>=1, x[1,1]>=1, x>=0)
-    @fact vexity(p) --> ConvexVexity()
+    @test vexity(p) == ConvexVexity()
     solve!(p)
-    @fact p.optval --> roughly(2, TOL)
-    @fact evaluate(sum(abs(x))) --> roughly(2, TOL)
+    @test p.optval ≈ 2 atol=TOL
+    @test evaluate(sum(abs(x))) ≈ 2 atol=TOL
   end
 
-  context("maximum atom") do
+  @testset "maximum atom" begin
     x = Variable(10)
-    a = rand(10, 1)
+    a = shuffle(collect(0.1:0.1:1.0))
```

Comment:
Why the switch to `shuffle` here?

Reply:
Using `rand` led to test failures with SCS solver for some cases. I think it happened because for some set of random numbers the problem might not be well conditioned (i.e. some numbers are too close to each other). So I changed it to a set of numbers from an equidistant grid but to still put some randomness there I `shuffle`d them.

Reply:
Alright. 

You should probably change it to `0.0:0.1:0.9` though as that is within the range of `rand` while `1.0` is not.

Reply:
For these tests the actual numbers don't really matter and they even don't need to be strictly within the range `[0, 1)`. If possible I would rather not touch them as they finally seem to be passing nicely.

Reply:
Alright sure